src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,60 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int NAME = 0;
+    private static final int AMOUNT = 1;
+    private static final String EXCEPTION_READ_TEXT = "Can't read the file";
+    private static final String EXCEPTION_WRITE_TEXT = "Can't writ to the file";
```

Comment:
There is a typo in the exception message string `EXCEPTION_WRITE_TEXT`. It should be "Can't write to the file" instead of "Can't writ to the file".